libs/backend/src/cast_vote_records/build_cast_vote_record.ts

```diff
@@ -160,7 +160,6 @@ type CVRContestRequiredBallotPageOptions =
     }
   | {
       ballotMarkingMode: 'hand';
-      // TODO: make this required when we separate image files from the CVR
```

Comment:
As far as I can tell, we still want to support the "no image file URI" case for "minimal" exports, i.e. VxCentralScan exports that include images only for ballots with write-ins. I might be missing some larger context, though, and am happy to bring this comment back, if so